src/containers/NetworkPopup/index.js

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import Materialize from "materialize-css";
+import PropTypes from "prop-types";
+import { reloadLocation } from "../../utils/helpers";
+import './NetworkPopup.scss';
+
+export class NetWorkPopup extends Component {
+  constructor(props) {
+    super(props);
+    this.ref = React.createRef();
+  }
+
+  onClick = () => {
+    reloadLocation();
+  };
+
+  getModal = () => this.ref.current;
+
+  componentDidMount() {
+    Materialize.Modal.init(this.getModal(), { dismissible: false });
+  }
+
+  componentDidUpdate() {
+    const { networkError } = this.props;
```

Comment:
Add tests for this functionality lines `25 - 30` i feel that they are important to your functionality

Reply:
<img width="1019" alt="screen shot 2018-11-29 at 16 35 09" src="https://user-images.githubusercontent.com/8180548/49225342-d66f7a80-f3f4-11e8-87a4-c3811cad490f.png">

@PaulKariukiRimiru an attempt to test it has proved futile. After a lot research and seeking of peer assistance I have not been able to come up with a test. See https://stackoverflow.com/questions/45833331/jest-can-not-deal-with-insertadjacentelement. This is a result of a `jsdom` shortcoming.